### PR TITLE
feat: Adding Inspector 2 Findings

### PR DESCRIPTION
**Pull request by andreaswittig**
Forward events about Inspector 2 Findings (see https://docs.aws.amazon.com/inspector/latest/user/eventbridge-integration.html).

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -363,8 +363,14 @@ variable "elastic_beanstalk_failed" {
...
+  type        = bool
+  description = "Get notified about Inspector 2 findings."
+  default     = true
+}
+
 variable "stage" {
   type        = string
   description = "marbot stage (never change this!)."
-  default     = "v1"
+  default     = "dev"
```
Comment (michaelwittig) [inline variables.tf]: please revert

File: variables.tf [hunk 2]
```diff
@@ -363,8 +363,14 @@ variable "elastic_beanstalk_failed" {
   default     = true
 }
 
+variable "inspector2_finding" {
+  type        = bool
+  description = "Get notified about Inspector 2 findings."
```
Comment (michaelwittig) [inline variables.tf]: Receive an alert, if an Inspector finding is created.

File: main.tf [hunk 3]
```diff
@@ -2572,3 +2572,30 @@ resource "aws_cloudwatch_event_target" "elastic_beanstalk_failed" {
...
+  description   = "Inspector 2 Finding (created by marbot)"
+  tags          = var.tags
+  event_pattern = <<JSON
+{
+  "source": [
+    "aws.inspector2"
+  ],
+  "detail-type": [
+    "Inspector2 Finding"
+  ]
```
Comment (michaelwittig) [inline main.tf]: do we need a filter for severity?

Comment (andreaswittig) [reply main.tf]: Possible severities are: `CRITICAL`, `HIGH`, and `MEDIUM`. Shall we ignore `MEDIUM` findings?

I thought about sending all findings to SNS/marbot and deliver them as alerts/notifications depending on the severity.

Comment (michaelwittig) [reply main.tf]: From the docs:
> Valid Values: INFORMATIONAL | LOW | MEDIUM | HIGH | CRITICAL | UNTRIAGED

I would start with `MEDIUM | HIGH | CRITICAL | UNTRIAGED`

Comment (andreaswittig) [reply main.tf]: Oh, seems like the EventBridge Docs for Inspector are not complete. :(

File: main.tf [hunk 4]
```diff
@@ -2572,3 +2572,41 @@ resource "aws_cloudwatch_event_target" "elastic_beanstalk_failed" {
...
+  ],
+  "detail": {
+    "severity": [
+      "MEDIUM",
+      "HIGH",
+      "CRITICAL",
+      "UNTRIAGED"
+    ],
+    "status": [
+      "ACTIVE"
```
Comment (andreaswittig) [inline main.tf:2599]: @michaelwittig I've added a filter for `status = ACTIVE`.

**Review by michaelwittig — APPROVED** [review]

**Pull request merged**